src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -94,6 +94,22 @@ def _add_model_args(parser):
         help=f"The source of the input data.",
     )
 
+    model_group.add_argument(
+        "--mean-input-tokens",
```

Comment:
Can we change these from
mean-input-tokens and stddev-input-tokens
to
input-tokens-mean and input-tokens-stddev
?

Reply:
I was copying llmperf arg name, do you just think yours are more readable?

Reply:
Actually, I would prefer ALL args related to input-data start with input-data* or something like that.

Reply:
and, these definitely don't belong in model_group. @dyastremsky is working on a proposal to clean up all of the arg groupings and order, but would be good to start from something that makes some sense.

Reply:
they are in desperate need of regrouping yes.

Reply:
updated